source/plugin.cpp

```diff
@@ -190,7 +200,7 @@ std::string own3d::get_api_endpoint(std::string_view const args)
 std::string own3d::get_web_endpoint(std::string_view const args /*= ""*/)
 {
 	constexpr std::string_view URL_ENDPOINT         = "https://own3d.pro/";
-	constexpr std::string_view URL_ENDPOINT_SANDBOX = "https://sandbox.own3d.pro/";
+	std::string_view           URL_ENDPOINT_SANDBOX = get_sandbox_url();
```

Comment:
std::string_view requires constant memory. std::string is deallocated before it is used, and will cause unexpected behavior.